pages/common/youtube-dl.md

```diff
@@ -1,6 +1,7 @@
 # youtube-dl
 
 > Download videos from YouTube and other websites.
+> Homepage: <http://rg3.github.io/youtube-dl/>.
```

Comment:
Prefer `https` if it is available :bowtie:

Reply:
@sbrl Is this linter (https://github.com/tldr-pages/tldr-lint) still used for validating tldr pages? If so this should be added as rule there...

Reply:
Did you mean to comment on this PR ? What has tldr-lint got to do with adding a homepage link ?

Reply:
In ideal world tldr-lint or any other linter used for checking tldr pages should report if http (home)page is linked and https version is available. It should report dead links also?

Reply:
Oh I see. I think that would put too much logic into the linter. I would want the heading text to be free-form without any strict format. The `Homepage:` format is just something we agreed to. I think it is not necessary to have linter checks for that.

Reply:
Maybe I wasn't clear enough. I wasn't talking about homepage specifically but all urls page may contain. If url is found and uses less secure scheme or points to dead end that should be reported...

Reply:
I see. I am neutral to it.

Reply:
Some commands' homepages may be only available over http and not https though.

Reply:
Quoting myself:
> if http (home)page is linked and https version is available.

That is relatively easy to check.

Reply:
This is probably not a place to discuss it but parsing tldr pages with bison is like killing fly with orbital ion cannon 🙄
After all tldr page is markdown document and there are plenty of markdown parsers around that can be adapted/wrapped to check page format. Such parser could provide hooks for specific page segment like urls which can in turn be used for http/https and/or dead link checking.

Reply:
Couldn't agree more. Sadly, that is the state of affairs now. If someone wants to rewrite the linter completely, they are more than welcome.